inventory_user/views/user_view.py

```diff
@@ -13,16 +13,26 @@
 from rest_framework.views import APIView
 
 from inventoryProject.permissions import IsStaffUser, IsSuperUser, IsSuperUserDelete, IsSuperUserOrStaffReadOnly
-from inventoryProject.utility.queryset_functions import get_or_not_found
+from inventoryProject.utility.queryset_functions import get_or_not_found, get_or_none
+from inventory_email_support.models import SubscribedManagers
 from inventory_user.serializers.user_serializer import UserSerializer, LargeUserSerializer
 from items.custom_pagination import LargeResultsSetPagination
 
+MEMBER_LIST = [SubscribedManagers]
 
 class InventoryUserList(generics.ListCreateAPIView):
     permission_classes = [IsSuperUserOrStaffReadOnly]
     queryset = User.objects.filter(is_active=True)
     serializer_class = UserSerializer
 
+    def post(self, request, *args, **kwargs):
+        if request.data.get('username') is not None:
+            user = get_or_none(User, username=request.data.get('username'))
+            if user is not None and not user.is_active:
+                raise MethodNotAllowed(method=self.post, detail="{user} has been marked inactive"

```

Comment:
can the comment be something like: An inactive user with this username already exists.